### ISSUE TITLE
Support aggregations over TIMESTAMP WITHOUT TIMEZONE fields?

### ISSUE THREAD

**Issue by jlowin** [hasura/graphql-engine#1217]
Hello,

At the moment it appears that aggregations over datetime fields (timestamps) are not supported -- querying the maximum timestamp, for example. Are there plans to add support for these pseudo-numeric types (or am I looking in the wrong place?)

Thanks!

**Comment by coco98** [comment]
@0x777 @rakeshkky Thoughts?

**Comment by 0x777** [comment]
Yes. We'll add support for this very soon.

**Comment by 0x777** [comment]
@jlowin This already works for timestamp and date columns. Can you give us the table schema where this doesn't work?

**Comment by jlowin** [comment]
@0x777 I apologize for the delayed reply. The columns that weren't working were typed as "timestamp without timezone"; when I changed them to "Timestamp" in the Hasura console, therefore converting them to "timestamp with timezone" (`timestampz`), it worked as described. Hasura's timestamp functionality has otherwise been perfect for these columns.

**Comment by CTOJoe** [comment]
I just ran into this issue. It does not seem to be resolved yet. If I have a column with type "timestamp without timezone". When querying the aggregate, the max and min is missing for the column. If I change the column to type "timestamp with timezone" the max and min aggregate is available. 

Here is the column I set to with timezone.
<img width="711" alt="Screen Shot 2020-01-06 at 1 38 34 PM" src="https://user-images.githubusercontent.com/2642064/71840225-d7102100-308a-11ea-8f41-0f68032fef89.png">

Here is the available aggregate fields
<img width="361" alt="Screen Shot 2020-01-06 at 1 39 26 PM" src="https://user-images.githubusercontent.com/2642064/71840440-5271d280-308b-11ea-944c-7eb1f303dc81.png">

Here is the type changed to "timestamp without time zone"
<img width="706" alt="Screen Shot 2020-01-06 at 1 39 48 PM" src="https://user-images.githubusercontent.com/2642064/71840478-63224880-308b-11ea-9234-385c7998492b.png">

The created_at aggregate disappears once the type is changed.
<img width="355" alt="Screen Shot 2020-01-06 at 1 40 37 PM" src="https://user-images.githubusercontent.com/2642064/71840556-877e2500-308b-11ea-9f87-a0f758954379.png">
